openspec/explorations/workspace-architecture.md

```
@@ -0,0 +1,681 @@
+openspec profile install <workflow>
+openspec profile uninstall <workflow>
+openspec profile list
+openspec profile show
+openspec config set delivery skills|commands|both
+openspec config get delivery
+openspec config list
+```
+8 subcommands, two concepts (profile + config)
+
+**After (simplified):**
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)